### ISSUE TITLE
Failed to set the session handler because headers have already been sent after upgrade to v3.4

### ISSUE THREAD

**Issue by umpirsky** [symfony/symfony#24800]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | yes
| RFC?             | no
| Symfony version  | v3.4.0-BETA2

Just upgraded from v3.3 to v3.4.0-BETA2. 

I have smoke tests for services which iterates them like:

```php
foreach ($container->getServiceIds() as $serviceId) {
    $service = $client->getContainer()->get($serviceId);
}
```

It errors with:

```
RuntimeException: Failed to set the session handler because headers have already been sent by "vendor/phpunit/phpunit/src/Util/Printer.php" at line 110.
    
vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php:402
vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php:114
```

May be related to https://github.com/symfony/symfony/issues/24524

**Comment by nicolas-grekas** [comment]
Yep, it's a duplicate.

**Comment by umpirsky** [comment]
I thought https://github.com/symfony/symfony/issues/24524 is PHP 7.2 specific. I am on PHP 7.1.6.

**Comment by javiereguiluz** [comment]
Closing as duplicate then. We'll reopen it if that's not true. Thanks!

**Comment by rcwsr** [comment]
I have this issue on php55 and php56

**Comment by rcwsr** [comment]
Seems to be fixed in symfony 2.8.30

**Comment by kala4ek** [comment]
Updated to 2.8.31 - works perfectly
